catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/operation/JdbcTableOperations.java

```diff
@@ -96,14 +97,17 @@ public void create(
   }
 
   @Override
-  public void drop(String databaseName, String tableName) throws NoSuchTableException {
-    LOG.info("Attempting to delete table {} from database {}", tableName, databaseName);
-    try (Connection connection = getConnection(databaseName)) {
-      JdbcConnectorUtils.executeUpdate(connection, generateDropTableSql(tableName));
+  public boolean drop(String databaseName, String tableName) {
+    try {
+      LOG.info("Attempting to delete table {} from database {}", tableName, databaseName);
+      dropTable(databaseName, tableName);
       LOG.info("Deleted table {} from database {}", tableName, databaseName);
-    } catch (final SQLException se) {
-      throw this.exceptionMapper.toGravitinoException(se);
+    } catch (NoSuchTableException e) {
+      return false;
+    } catch (GravitinoRuntimeException e) {
+      throw new RuntimeException("Failed to drop table " + tableName + " from " + databaseName, e);
```

Comment:
The wrap seems unnecessary

Reply:
This comment has not been addressed

Reply:
> This comment has not been addressed

If the wrapping is not used , how can one capture the `NoSuchSchemaException` thrown from `exceptionMapper.toGravitinoException(se); `?

Reply:
Hasn't this already been done in [dropTable](https://github.com/datastrato/gravitino/pull/3222/files/cf47992aab40d83c5ee74cfdec719f1b7323827e#diff-866388a5946d73deb4da7ec949f9be59eda658fe611c5a13d43f719484ae9864R266-R272)?